techniques/aria/ARIA11.html

```diff
@@ -22,6 +22,8 @@
 </li>
          <li>navigation: A collection of links suitable for use when navigating the document or related documents.
 </li>
+         <li>region: A region of content for an author-specific purpose that is important for users to be able to navigate to. It maps to <code>section</code> in HTML5, and needs a label to be included as a landmark for assistive technology.</li>
```

Comment:
Does this document still refer to HTML5? I think most W3C documents are just saying "HTML" now?

```suggestion
         <li>region: A region of content for an author-specific purpose that is important for users to be able to navigate to. It maps to <code>section</code> in HTML, and needs a label to be included as a landmark for assistive technology.</li>
```

Reply:
Actually, please consider changing "a label" to "an accessible label" so that it isn't confused with "a <label>":

```suggestion
         <li>region: A region of content for an author-specific purpose that is important for users to be able to navigate to. It maps to <code>section</code> in HTML5, and needs an accessible label to be included as a landmark for assistive technology.</li>
```